.internal-ci/helm/mc-core-common-config/templates/_helpers.tpl

```diff
@@ -51,21 +51,3 @@ Selector labels
 app.kubernetes.io/name: {{ include "mcCoreCommonConfig.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
-
```

Comment:
remove mobilecoind stuff from mc-core-common-config.  Pretty close to deprecating this whole chart.